MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
+        /// </summary>
+        public class Random
```

Comment:
+1 for not having it as a nested class.  If it's a nested class there's no
way of pushing it into the global namespace with a "using" block AFAIK.

On Wed, Jan 6, 2016, 02:55 KonajuGames <notifications@github.com> wrote:

> In MonoGame.Framework/Random.cs
> <https://github.com/mono/MonoGame/pull/3789#discussion_r48859224>:
>
> > @@ -0,0 +1,386 @@
> > +﻿using System;
> > +using System.Collections.Generic;
> > +using System.Linq;
> > +using System.Text;
> > +
> > +namespace Microsoft.Xna.Framework
> > +{
> > +    public static partial class MathHelper
> > +    {
> > +        /// <summary>
> > +        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
> > +        /// </summary>
> > +        public class Random
>
> The source file should have the standard MonoGame header block. See
> https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Utilities/Hash.cs#L1
> for an example.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/MonoGame/pull/3789/files#r48859224>.
>